### PR TITLE
Replaced relative link with absolute links and remove locale

### PR DESCRIPTION
**Pull request by sima-zhu**
The PR is to turn on the link check flag.

1. Disable the relative links.
1. Remove locale "en-us" from link.

Pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=541658&view=results

### CODE REVIEW

File: sdk/eventgrid/Azure.Messaging.EventGrid/README.md [hunk 1]
```diff
@@ -187,18 +187,18 @@ foreach (CloudEvent cloudEvent in cloudEvents)
             break;
         case "Microsoft.Storage.BlobDeleted":
             // Example for deserializing system events using GetData<T>
-            StorageBlobDeletedEventData blobDeleted = cloudEvent.GetData<StorageBlobDeletedEventData>();
+            StorageBlobDeletedEventData blobDeleted = cloudEvent.GetData<StorageBlobDeletedEventData>(http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventgrid/Azure.Messaging.EventGrid/README.md);
```
Comment (pakrym) [inline sdk/eventgrid/Azure.Messaging.EventGrid/README.md:190]: This seems very wrong

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -461,7 +461,7 @@ Much of the management plane SDK code is generated from metadata specs about the
 
 ## Versioning
 
-For more information on how we version see [Versioning](doc/dev/Versioning.md)
+For more information on how we version see [Versioning](http://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md)
```
Comment (jsquire) [inline CONTRIBUTING.md:464]: nit:  I believe this should be `https`.

File: README.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ Documentation and code samples for these libraries can be found [here](https://a
 Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) <secure@microsoft.com>. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://www.microsoft.com/msrc/faqs-report-an-issue).
 
 ## Contributing
-For details on contributing to this repository, see the [contributing guide](CONTRIBUTING.md).
+For details on contributing to this repository, see the [contributing guide](http://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).
```
Comment (jsquire) [inline README.md:64]: nit:  I believe this should be `https`

File: doc/README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 This folder contains some documentations for this repository:
 
 The folder structure is the following
-- [dev](./dev) : contains advanced documentation for _developers_ of SDK (not _consumers_ of SDK)
-- [ApiDocGeneration](./ApiDocGeneration) : contains scripts and assets for generating API documentation
+- [dev](http://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev) : contains advanced documentation for _developers_ of SDK (not _consumers_ of SDK)
```
Comment (jsquire) [inline doc/README.md:5]: nit:  I believe these should all be `https`

File: doc/dev/Track2TestFramework.md [hunk 5]
```diff
@@ -1 +1 @@
-See the project [readme](../../sdk/core/Azure.Core.TestFramework/README.md).
\ No newline at end of file
+See the project [readme](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/README.md).
```
Comment (jsquire) [inline doc/dev/Track2TestFramework.md:1]: nit:  I believe this should be `https`

File: sdk/eventgrid/Azure.Messaging.EventGrid/README.md (hunk 1 above)
Comment (weshaggard) [reply sdk/eventgrid/Azure.Messaging.EventGrid/README.md:190]: Yes this is definitely not correct. @sima-zhu looks like a find/replace `()` is doing the wrong thing.

File: sdk/appconfiguration/Azure.Data.AppConfiguration/README.md [hunk 6]
```diff
@@ -218,13 +218,13 @@ Message: Azure.RequestFailedException : StatusCode: 404, ReasonPhrase: 'Not Foun
...
-- [Make a configuration setting readonly](samples/Sample3_SetClearReadOnly.md): Make a configuration setting read-only, and then return it to a read-write state.
-- [Read revision history](samples/Sample4_ReadRevisionHistory.md): Read the revision history of a configuration setting that has been changed.
-- [Get a setting if changed](samples/Sample5_GetSettingIfChanged.md): Save bandwidth by using a conditional request to retrieve a setting only if it is different from your local copy.
-- [Update a setting if it hasn't changed](samples/Sample6_UpdateSettingIfUnchanged.md): Prevent lost updates by using optimistic concurrency to update a setting only if your local updates were applied to the same version as the resource in the configuration store.
-- [Create a mock client](samples/Sample7_MockClient.md): Mock a client for testing using the [Moq library][moq].
+- [Hello world](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample1_HelloWorld.md): Create and delete a configuration setting.
```
Comment (jsquire) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/README.md:221]: nit: I believe these all should be `https`

File: sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample1_HelloWorld.md [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
 
  ## Create a ConfigurationClient
  
-To interact with Azure App Configuration, you need to instantiate a `ConfigurationClient`. You can use either an endpoint URL and a [`TokenCredential`](../../../identity/Azure.Identity/README.md#credentials) or a connection string.
+To interact with Azure App Configuration, you need to instantiate a `ConfigurationClient`. You can use either an endpoint URL and a [`TokenCredential`](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md#credentials) or a connection string.
```
Comment (jsquire) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample1_HelloWorld.md:7]: nit: I believe this should be `https`

File: sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample2_HelloWorldExtended.md [hunk 8]
```diff
@@ -1,10 +1,10 @@
...
+This sample demonstrates how to send requests to the Azure App Configuration service asynchronously. It also shows how to create and retrieve configuration settings with labels, and provides an example scenario where labels are used to group configuration settings for "beta" and "production" application instances.  To get started, you'll need a connection string to Azure App Configuration. See the [README](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/README.md) for links and instructions.
 
  ## Create a ConfigurationClient
  
-To interact with Azure App Configuration, you need to instantiate a `ConfigurationClient`. You can use either an endpoint URL and a [`TokenCredential`](../../../identity/Azure.Identity/README.md#credentials) or a connection string.
+To interact with Azure App Configuration, you need to instantiate a `ConfigurationClient`. You can use either an endpoint URL and a [`TokenCredential`](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md#credentials) or a connection string.
```
Comment (jsquire) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample2_HelloWorldExtended.md:7]: nit: I believe this should be `https`

File: sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample3_SetClearReadOnly.md [hunk 9]
```diff
@@ -1,10 +1,10 @@
 # Make a Configuration Setting Read-Only
 
-This sample demonstrates how to make a configuration value read-only and return it back to the read-write state, as well as the exception that is thrown if a user attempts to write to a read-only setting. To get started, you'll need a connection string to the Azure App Configuration. See the [README](../README.md) for links and instructions.
+This sample demonstrates how to make a configuration value read-only and return it back to the read-write state, as well as the exception that is thrown if a user attempts to write to a read-only setting. To get started, you'll need a connection string to the Azure App Configuration. See the [README](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/README.md) for links and instructions.
```
Comment (jsquire) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample3_SetClearReadOnly.md:3]: nit: I believe these should be `https`

File: sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample4_ReadRevisionHistory.md [hunk 10]
```diff
@@ -2,11 +2,11 @@
 
 This sample demonstrates how to read the revision history of a Configuration Setting in a Configuration Store. To do this, we create a setting, change it twice to create revisions, then read the revision history for that setting using a `SettingSelector` that uniquely identifies the configuration setting by its key name. The sample uses a configuration setting with a timestamp in the key name to ensure the setting hasn't been used before and thereby minimize the size of the revision history.
  
-To get started, you'll need a connection string to the Azure App Configuration. See the [README](../README.md) for links and instructions.
+To get started, you'll need a connection string to the Azure App Configuration. See the [README](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/README.md) for links and instructions.
```
Comment (jsquire) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample4_ReadRevisionHistory.md:5]: nit: I believe these should be `https`

File: sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample5_GetSettingIfChanged.md [hunk 11]
```diff
@@ -26,7 +26,7 @@ public static ConfigurationSetting GetConfigurationSettingIfChanged(Configuratio
 
 ## Create a ConfigurationClient
 
-To interact with Azure App Configuration, you need to instantiate a `ConfigurationClient`. You can use either an endpoint URL and a [`TokenCredential`](../../../identity/Azure.Identity/README.md#credentials) or a connection string.
+To interact with Azure App Configuration, you need to instantiate a `ConfigurationClient`. You can use either an endpoint URL and a [`TokenCredential`](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md#credentials) or a connection string.
```
Comment (jsquire) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample5_GetSettingIfChanged.md:29]: nit: I believe this should be `https`

File: sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample6_UpdateSettingIfUnchanged.md [hunk 12]
```diff
@@ -4,7 +4,7 @@ This sample illustrates how to update a setting in the configuration store only
 
 In a hypothetical scenario, we release several virtual machines from our application's resource pool, and update the `available_vms` configuration setting to reflect this. In this scenario, if another client were to have modified `available_vms` since we last retrieved it, and we updated it unconditionally, our update would overwrite their changes and the resulting value of `available_vms` would be incorrect. We show in the sample how to implement optimistic concurrency to apply the update in a way that doesn't overwrite other clients' changes.
 
-To get started, you'll need a connection string to the Azure App Configuration. See the [README](../README.md) for links and instructions.
+To get started, you'll need a connection string to the Azure App Configuration. See the [README](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/README.md) for links and instructions.
```
Comment (jsquire) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample6_UpdateSettingIfUnchanged.md:7]: nit: I believe these should be `https`

File: sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample7_MockClient.md [hunk 13]
```diff
@@ -1,10 +1,10 @@
 # Mock a client for testing using the Moq library
 
-This sample illustrates how to use [Moq](https://github.com/Moq/moq4/) to create a unit test that mocks the response from a ConfigurationClient method. For more examples of mocking, see the [Azure.Data.AppConfiguration.Tests](../tests/ConfigurationMockTests.cs) project.
+This sample illustrates how to use [Moq](https://github.com/Moq/moq4/) to create a unit test that mocks the response from a ConfigurationClient method. For more examples of mocking, see the [Azure.Data.AppConfiguration.Tests](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationMockTests.cs) project.
```
Comment (jsquire) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample7_MockClient.md:3]: nit: I believe these should be `https`

File: sdk/appconfiguration/CONTRIBUTING.md [hunk 14]
```diff
@@ -58,21 +58,21 @@ Our samples are structured as unit tests so we can easily verify they're up to d
 
 ## Development history
 
-For additional insight and context, the development, release, and issue history for the Azure Event Hubs client library is available in read-only form, in its previous location, the [Azure App Configuration .NET repository]().
+For additional insight and context, the development, release, and issue history for the Azure Event Hubs client library is available in read-only form, in its previous location, the [Azure App Configuration .NET repository](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/CONTRIBUTING.md).
```
Comment (jsquire) [inline sdk/appconfiguration/CONTRIBUTING.md:61]: nit: I believe these should be `https`

File: sdk/appconfiguration/README.md [hunk 15]
```diff
@@ -13,8 +13,8 @@ This project welcomes contributions and suggestions. Most contributions require
 This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
 <!-- LINKS -->
-[app_config]: ./Azure.Data.AppConfiguration/README.md
-[contrib]: ./CONTRIBUTING.md
+[app_config]: http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/README.md
```
Comment (jsquire) [inline sdk/appconfiguration/README.md:16]: nit: I believe these should be `https`

**Review by drwill-ms — APPROVED** [review]
looks good for ADT

File: sdk/core/Azure.Core/README.md [hunk 16]
```diff
@@ -66,7 +66,7 @@ SecretClientOptions options = new SecretClientOptions()
 SecretClient client = new SecretClient(new Uri("http://example.com"), new DefaultAzureCredential(), options);
 ```
 
-More on client configuration in [client configuration samples](samples/Configuration.md)
+More on client configuration in [client configuration samples](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/Configuration.md)
```
Comment (jsquire) [inline sdk/core/Azure.Core/README.md:69]: nit: I believe these should be `https`

File: sdk/core/Azure.Core/samples/Diagnostics.md [hunk 17]
```diff
@@ -70,7 +70,7 @@ SecretClientOptions options = new SecretClientOptions()
 
 ### ASP.NET Core applications
 
-If your are using Azure SDK libraries in ASP.NET Core application consider using the `Microsoft.Extensions.Azure` package that provides integration with `Microsoft.Extensions.Logging` library. See [Microsoft.Extensions.Azure readme](../../../extensions/Microsoft.Extensions.Azure/README.md) for more details.
+If your are using Azure SDK libraries in ASP.NET Core application consider using the `Microsoft.Extensions.Azure` package that provides integration with `Microsoft.Extensions.Logging` library. See [Microsoft.Extensions.Azure readme](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/extensions/Microsoft.Extensions.Azure/README.md) for more details.
```
Comment (jsquire) [inline sdk/core/Azure.Core/samples/Diagnostics.md:73]: nit: I believe this should be `https`

File: sdk/core/Azure.Core/samples/Pipeline.md [hunk 18]
```diff
@@ -9,7 +9,7 @@ Before request is sent to the service it travels through the pipeline which cons
 Azure SDKs provides a way to add policies to the pipeline at two positions:
 
  - per-call policies get executed once per request
- - per-retry policies get executed every time request is retried, see [Retries samples](./Configuration.md#configuring-retry-options) for how to configure retries.
+ - per-retry policies get executed every time request is retried, see [Retries samples](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/Configuration.md#configuring-retry-options) for how to configure retries.
```
Comment (jsquire) [inline sdk/core/Azure.Core/samples/Pipeline.md:12]: nit: I believe this should be `https`

File: sdk/core/Azure.Core/samples/Response.md [hunk 19]
```diff
@@ -7,7 +7,7 @@ Most client methods return one of the following types:
  - `Response<T>` -  a value and HTTP response
  - `Pageable<T>` -  a collection of values retrieved in pages
  - `AsyncPageable<T>` - a collection of values asyncrounosly retrieved in pages
- - `*Operation<T>` - a long-running operation see [long running operation samples](LongRunningOperations.md)
+ - `*Operation<T>` - a long-running operation see [long running operation samples](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/LongRunningOperations.md)
```
Comment (jsquire) [inline sdk/core/Azure.Core/samples/Response.md:10]: nit: I believe this should be `https`

File: sdk/servicebus/Azure.Messaging.ServiceBus/README.md [hunk 20]
```diff
@@ -12,15 +12,15 @@ Use the client library for Azure Service Bus to:
 
 - Implement complex workflows: message sessions support scenarios that require message ordering or message deferral.
 
-[Source code](.) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Messaging.ServiceBus/) | [API reference documentation](https://azure.github.io/azure-sdk-for-net/servicebus.html) | [Product documentation](https://docs.microsoft.com/en-us/azure/service-bus/) | [Migration guide](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md)
+[Source code](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Azure.Messaging.ServiceBus/) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Messaging.ServiceBus/) | [API reference documentation](https://azure.github.io/azure-sdk-for-net/servicebus.html) | [Product documentation](https://docs.microsoft.com/azure/service-bus/) | [Migration guide](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md)
```
Comment (jsquire) [inline sdk/servicebus/Azure.Messaging.ServiceBus/README.md:15]: nit: I believe these should be `https`

**Review by jsquire — COMMENTED** [review]
I believe that all the `http` links should be `https`;  since the pattern seems to not be isolated to any area, I stopped leaving individual comments about 1/2 through scanning.

File: sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md [hunk 21]
```diff
@@ -10,7 +10,7 @@ The Event Processor client library is a companion to the Azure Event Hubs client
 
 - Managing checkpoints and state for processing in a durable manner using Azure Storage blobs as the underlying data store.
 
-[Source code](.) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Messaging.EventHubs.Processor/) | [API reference documentation](https://aka.ms/azsdk-dotnet-eventhubs-processor-docs) | [Product documentation](https://docs.microsoft.com/azure/event-hubs/)
+[Source code](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs.Processor/) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Messaging.EventHubs.Processor/) | [API reference documentation](https://aka.ms/azsdk-dotnet-eventhubs-processor-docs) | [Product documentation](https://docs.microsoft.com/azure/event-hubs/)
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md:13]: nit: I believe these should be `https`

File: sdk/eventhub/Azure.Messaging.EventHubs.Shared/README.md [hunk 22]
```diff
@@ -1,6 +1,6 @@
...
+# Azure Event Hubs code sharing library
 
-This library is intended to serve as a common location for non-public code that is shared amongst the packages that comprise the [Azure Event Hubs client library for .NET](./../Azure.Messaging.EventHubs) development ecosystem.  The artifacts in this library are intended to be small and cohesive units of work with minimal dependencies, which offer common building blocks for internal implementation.  These artifacts should be accompanied by their suite of unit tests, allowing the `Shared` library to have sole responsibility for them.
+This library is intended to serve as a common location for non-public code that is shared amongst the packages that comprise the [Azure Event Hubs client library for .NET](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs) development ecosystem.  The artifacts in this library are intended to be small and cohesive units of work with minimal dependencies, which offer common building blocks for internal implementation.  These artifacts should be accompanied by their suite of unit tests, allowing the `Shared` library to have sole responsibility for them.
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs.Shared/README.md:3]: nit: I believe these should be `https`

File: sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md [hunk 23]
```diff
@@ -234,9 +234,9 @@ _(A community contribution, courtesy of [albertodenatale](https://github.com/alb
 
 - A large portion of the public API surface, including members and parameters, have had adjustments to their naming in order to improve discoverability, provide better context to developers, and better conform to the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html).
 
-- The `EventProcessorClient` has been moved into its own [package](./../Azure.Messaging.EventHubs.Processor) and evolved into an opinionated implementation on top of Azure Storage Blobs.  This is intended to offer a more streamlined developer experience for the majority of scenarios and allow developers to more easily take advantage of the processor.
+- The `EventProcessorClient` has been moved into its own [package](http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs.Processor) and evolved into an opinionated implementation on top of Azure Storage Blobs.  This is intended to offer a more streamlined developer experience for the majority of scenarios and allow developers to more easily take advantage of the processor.
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md:237]: nit: I believe these should be `https`

File: sdk/eventhub/Microsoft.Azure.EventHubs/README.md [hunk 24]
```diff
@@ -13,7 +13,7 @@ computation and filtering. Processing may also involve distribution or storage o
 Event Hub consumers are often robust and high-scale platform infrastructure parts with built-in analytics capabilities, like Azure 
 Stream Analytics, Apache Spark, or Apache Storm.  
 
-This directory contains the open source subset of the .NET SDK. For documentation of the complete Azure SDK, please see the [Microsoft Azure .NET Developer Center](http://azure.microsoft.com/en-us/develop/net/).
+This directory contains the open source subset of the .NET SDK. For documentation of the complete Azure SDK, please see the [Microsoft Azure .NET Developer Center](http://azure.microsoft.com/develop/net/).
```
Comment (jsquire) [inline sdk/eventhub/Microsoft.Azure.EventHubs/README.md:16]: nit: I believe this should be `https`

File: CONTRIBUTING.md (hunk 2 above)
Comment (sima-zhu) [reply CONTRIBUTING.md:464]: Sure. Found the issue in java PR tool. Will fix it. Thanks!

**Comment by sima-zhu** [comment]
New pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=541883&view=logs&j=10327055-d4d6-50a6-a312-3d2cd4bd9cb9&t=65f0e4aa-f456-5911-deca-bd807ca3354c

File: sdk/eventgrid/Azure.Messaging.EventGrid/README.md (hunk 1 above)
Comment (sima-zhu) [reply sdk/eventgrid/Azure.Messaging.EventGrid/README.md]: Fixed it.

File: sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample1_RbacHelloWorldAsync.md [hunk 25]
```diff
@@ -75,4 +75,4 @@ KeyVaultRoleAssignment deletedAssignment = await client.DeleteRoleAssignmentAsyn
 
 <!-- LINKS -->
 [azure_cli]: https://docs.microsoft.com/cli/azure
-[DefaultAzureCredential]: ../../../identity/Azure.Identity/README.md
+[DefaultAzureCredential]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample1_RbacHelloWorldAsync.md:78]: While you're at it, this is what it should be:

```suggestion
[DefaultAzureCredential]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md#defaultazurecredential
```

File: sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample1_RbacHelloWorldSync.md [hunk 26]
```diff
@@ -67,5 +67,5 @@ KeyVaultRoleAssignment deletedAssignment = client.DeleteRoleAssignment(KeyVaultR
 
 <!-- LINKS -->
 [azure_cli]: https://docs.microsoft.com/cli/azure
-[DefaultAzureCredential]: ../../../identity/Azure.Identity/README.md
+[DefaultAzureCredential]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample1_RbacHelloWorldSync.md:70]:
```suggestion
[DefaultAzureCredential]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md#defaultazurecredential
```

**Review by heaths — APPROVED** [review]
Key Vault and Search LGTM. I made a couple of suggestions - no problems with your changes, but might be good to improve the links while you're making edits.

**Review by weshaggard — APPROVED** [review]

**Comment by weshaggard** [comment]
@sima-zhu lets be sure to file a follow-up issue to convert all http links to https.

**Pull request merged**